docs/how-to/ownership.mdx

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 7
+title: Modeling Ownership
+---
+
+Ownership plays an important role in managing and securing access to resources - this guide explains how to setup ownership using Permit
+
+## When Do We Need Ownership?
+
+Ownership becomes crucial in scenarios where control over data, files, or any digital assets is essential to ensure that only authorized individuals or entities can access, modify, or share them.
+
+There are two primary methods that can be employed to model ownership of resources -
+Attribute Based Access Control (ABAC) and Relationship Based Access Control (ReBAC).
+Both approaches offer unique advantages and cater to different requirements for managing access controls.
+
+Let's take a look at a simple example:
+
+:::tipExample
```

Comment:
I think it is better to put it in a quote or code style. The current admonition looks weird, especially because of the next one with the RBAC.

Reply:
Disagree.